### DISCUSSION TITLE
How to get the location of a variable name in the whole function?

### DISCUSSION THREAD

**Discussion by skye95git** [tree-sitter/py-tree-sitter#233] [Q&A]
I want to use your tool to anonymize function variable names. I tried to use your tool to parse the function, and found that after parsing, the variable names all appear as identifiers. In this case, how to distinguish different variable names and where they appear in the function body?

For example, the original code:
```python
def find_max(a, b):
    if a > b:
        return a
    else:
        return b
```

The result of `tree.root_node.sexp()`:
```
(module 
  (function_definition name:   
  (identifier) parameters: (parameters (identifier) (identifier)) 
  body: (block (if_statement condition: (comparison_operator (identifier) (identifier)) consequence: (block (return_statement (identifier))) alternative: (else_clause body: (block (return_statement (identifier))))))))
```

But I want to get:
```python
def find_max(var_0, var_1):
    if var_0 > var_1:
        return var_0
    else:
        return var_1
```

**Comment by MartV0** [comment]
You can grab the text content of an identifier using `node.text.decode('utf-8')`.
